win32ss/user/user32/misc/dllmain.c

```diff
@@ -439,6 +439,7 @@ Init(PUSERCONNECT UserCon /*PUSERSRV_API_CONNECTINFO*/)
         gHandleEntries = SharedPtrToUser(gHandleTable->handles);
         gSharedInfo = UserCon->siClient;
         gSharedInfo.psi = gpsi;
+        gSharedInfo.aheList = gHandleTable;
```

Comment:
Why would this not be set correctly in `siClient` ?

Reply:
Especially since it's taken from
```c
gHandleTable = SharedPtrToUser(UserCon->siClient.aheList);
gSharedInfo = UserCon->siClient;
```

Reply:
I cannot understand why `SharedPtrToUser` is needed.

Reply:
This aspect will be fixed by PR #3937 very soon.